### DISCUSSION TITLE
Ohno

### DISCUSSION THREAD

**Discussion by dangxuantruong10a13** [dangxuantruong10a13/nhom9_10a13#12] [Announcements]
Các bạn quên bỏ link nguồn vào mấy phần lý thuyết rầu nè, gửi mình các link các phần ở đây để mình thêm vào nek

**Comment by TranBaoLam10C13** [comment]
https://vietjack.com/giai-bai-tap-tin-hoc-10/ly-thuyet-bai-toan-va-thuat-toan.jsp

**Reply by dangxuantruong10a13** [reply]
mình gắn zô mấy chỗ mà thấy giống với cái này nke

**Reply by TranBaoLam10C13** [reply]
Ghi ở cuối dc r

**Comment by NgHan12pn** [comment]
https://loigiaihay.com/cau-5-trang-44-sgk-tin-hoc-10-c156a24885.html
